src/controllers/stallController.ts

```diff
@@ -215,9 +207,9 @@ async function destroyStallImages(req: Request, res: Response, next: NextFunctio
 }
 
 export const findStallsByIdsFunc = findStallsByIds;
-export const mapStallToCardFunc = mapStallToCard;
+export const findAllStallsFunc = indexStall;
 
-export const indexStallFuncs = [indexStall];
+export const indexStallFuncs = [indexStall, showStalls];
```

Comment:
Are you just shifting `res.status(200).json(stalls);` to a new middleware?

Reply:
No need to keep using middlewares, can also use normal functions -> middlewares are harder to extend as they require passing information through `req`